### ISSUE TITLE
Validate an DOCX Report that Uses Velocity Syntax

### ISSUE THREAD

**Issue by msalamon1** [opensagres/xdocreport#110]
Is there any way to pre-validate a docx report that uses the Velocity syntax.  Basically, I'm trying to catch Velocity syntax errors before trying to generate a document and get a report on where they exist.  In particular, errors adding the velocity code to the merge fields, conditions and loops without closing statements, etc.

**Comment by angelozerr** [comment]
Today you must generate a report to throw Velocity syntax (the first velocity syntax). Is it not enough?

**Comment by msalamon1** [comment]
Yes and no. :)   I think what I meant to ask is whether there is any way to get more information to help identify the error.  Most of the errors that are generated are Velocity ParserErrorExceptions and there is very little information provided.

Column number, line number and invalid syntax are either null or don't provide anything useful in the context of a docx report.  I wasn't sure if there was another way to help identify the source of the error.
